battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/play/Playing.java

```diff
@@ -54,15 +56,15 @@ public ShootResponse shoot(@RequestParam(name = "fieldNumber") int fieldNumber)
     }
 
     @RequestMapping(method = RequestMethod.GET, path = "/turnstatus")
-    public TurnStatusResponse turnStatus(HttpSession session){
+    public TurnStatus turnStatus(HttpSession session){
         // this method will call iterally, so i dont think that logging is a good idea
```

Comment:
* When I means myself it should ALWAYS be capitalized. 
* iterally? Either iteratively or literally, I think?
* also, if this is a method comment, why not Javadoc?